apache/superset
# fix(explore): simple tab content input problem in the filter control

### SUMMARY

This PR fixes the simple tab content input problem. Currently the content cursor jumps to the end after input. See the video.
### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF


### before

https://user-images.githubusercontent.com/11830681/147406524-c92af829-e7ee-408d-89ca-fcd4a73f9ded.mov


### after

https://user-images.githubusercontent.com/11830681/147406527-342834d5-15d5-4b81-88fa-9f7250a22f6f.mov


### TESTING INSTRUCTIONS

1. Go to explore
2. Create a filter
3. Enter in filter value
4. See the error

### ADDITIONAL INFORMATION
